corehq/apps/locations/models.py

```diff
@@ -379,12 +379,12 @@ class SQLLocation(AdjListModel):
     domain = models.CharField(max_length=255, db_index=True)
     name = models.CharField(max_length=255, null=True)
     location_id = models.CharField(max_length=100, db_index=True, unique=True)
-    location_type = models.ForeignKey(LocationType, on_delete=models.CASCADE)
+    location_type = models.ForeignKey(LocationType, on_delete=models.CASCADE, db_index=True)
```

Comment:
I don't see a migration for this but I also think that foreign keys are automatically indexed so this shouldn't be necessary

Reply:
hm ok. i will just delete that addition